src/store/setToolMode.js

```diff
@@ -236,21 +229,41 @@ function setToolMode (mode, changeEvent, toolName, options, isTouchActive) {
 }
 
 /**
+ * Find tool's that conflict with the incoming tool's mouse/touch bindings and
+ * resolve those conflicts.
  *
+ * @param {*} element
+ * @param {*} tool
+ * @param {*} options
+ * @param {*} isTouchActive
+ * @private
+ */
+function _resolveInputConflicts (element, tool, options, isTouchActive) {
+  // There are two API overloads for changing a tool's mode. This helps us
+  // Determine where to find our mask values.
+  const isValueApiOverload = typeof options === 'number';
```

Comment:
This is a much cleaner and easier to follow imo 👍 .

The methods `_resolveMouseInputConflicts` and `_resolveTouchInputConflicts` are easy to parse also.